File: tests/integration/scripts/echovars.py
#!/usr/bin/env python3
import os
import sys
import time
import tempfile

# get list of variables for CLF
clf = [f"{v}=<{os.environ.get(v)}>" for v in os.environ if v.startswith("CLF")]

# build file into temporary directory with the first argument as the file name
tmpfile = os.path.join(sys.argv[1])
output = open(tmpfile, "a+")
output.write(f"{sys.argv[1]}-{clf}\n")

sys.exit(100)

File: tests/integration/scripts/echovars.rb
# first arg is the name of the output file
output_file = ARGV[0]

# get list of variables for CLF
clf = ENV.select {|k,v| k.start_with?("CLF_")}
pid = Process.pid

# build file into temporary directory with the first argument as the file name
output = File.open(output_file, "a")
output.write("#{pid}-#{output_file}-#{clf}\n")

exit(100)